cli/cmd/compose/convert_kube.go

```diff
@@ -0,0 +1,31 @@
+// +build kube
+
+/*
+   Copyright 2020 Docker Compose CLI authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package compose
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	addFlagsFuncs = append(addFlagsFuncs, func(cmd *cobra.Command, opts *convertOptions) {
+		flags := cmd.Flags()
+		flags.StringVar(&opts.OutputDir, "output-dir", "", "Save to directory")
```

Comment:
Let's call this --output , and reuse the option for specifying either a file output or a folder output in the case of kube, rather than add several options for different backends

Reply:
Actually adding this on local compose & ecs will avoid this thing for adding flags only for hidden backends. (ACI does not have the convert command at all)